### PR TITLE
[master < T576-FL] Update pull request template

### PR DESCRIPTION
**Pull request by antoniofilipovic**
[master < Task] PR
- [x] Check, and update documentation if necessary
- [x] Provide the full content or a guide for the final git message

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 - [ ] Provide the full content or a guide for the final git message
 
 [master < Task] PR
 - [ ] Check, and update documentation if necessary
-- [ ] Update [changelog](https://docs.memgraph.com/memgraph/changelog)
 - [ ] Provide the full content or a guide for the final git message
+
+
+To keep docs changelog up to date, one more thing to do:
+- [ ] Write a release note here
```
Comment (vpavicic) [inline .github/pull_request_template.md:13]: and tag me (docs) to check

Comment (antoniofilipovic) [reply .github/pull_request_template.md:13]: Where to tad you? In the slack channel or here on github in comments at the end somewhere?

Comment (vpavicic) [reply .github/pull_request_template.md:13]: I think the shortest path is to do it on github in the comments and we can continue the discussion about the note in the comments

**Review by Josipmrden — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (antoniofilipovic) [reply .github/pull_request_template.md:13]: @vpavicic added this part

**Review by vpavicic — APPROVED** [review]

**Pull request merged**